src/fromager/__main__.py

```diff
@@ -6,13 +6,30 @@
 
 import click
 
-from . import clickext, commands, constraints, context, overrides, settings
+from . import (
+    clickext,
+    commands,
+    constraints,
+    context,
+    external_commands,
+    overrides,
+    settings,
+)
 
 logger = logging.getLogger(__name__)
 
 TERSE_LOG_FMT = "%(message)s"
 VERBOSE_LOG_FMT = "%(levelname)s:%(name)s:%(lineno)d: %(message)s"
 
+try:
+    external_commands.supports_network_isolation()
```

Comment:
I don't see a function with this name in this PR.

Reply:
Another refactoring going wrong. Naming is hard.